### DISCUSSION TITLE
Support for Jetbrains gateway

### DISCUSSION THREAD

**Discussion by chattarajoy** [GoogleCloudPlatform/cloud-code-intellij#3056] [General]
Are there any plans to support this on Jetbrains gateway? 

<img width="1412" alt="image" src="https://user-images.githubusercontent.com/8450903/155319250-03260f4f-c006-4eb7-92e7-fbcdc0448d9a.png">

**Comment by etanshaul** [comment]
Hey @chattarajoy yes, it is on our radar. We haven't yet done the investigation to see what it would take to support it properly. I am going to migrate this to an issue to track the effort and provide updates when we have them. Thanks for the request.
